src/SFA.DAS.PSRService.Web/Views/Report/Summary.cshtml

```diff
@@ -23,98 +27,98 @@
                 <h1 class="heading-xlarge">Review details</h1>
             </legend>
         }
-            <p id="report-summary-no-reports-message">No reports have been created yet.</p>
+        <p id="report-summary-no-reports-message">No reports have been created yet.</p>
     }
     else
     {
-            <div class="column-full review-details">
-                @if (!Model.Report.Submitted)
+        <div class="column-full review-details">
+            @if (!Model.Report.Submitted)
+            {
+                if (Model.CanBeEdited)
                 {
-                    if (Model.CanBeEdited)
-                    {
-                        <a class="link-back" asp-controller="Report" asp-action="Edit">Back</a>
-                    }
-                    else
-                    {
-                        <a class="link-back" asp-controller="Home" asp-action="Index">Back</a>
-                    }
-
+                    <a class="link-back" asp-controller="Report" asp-action="Edit">Back</a>
                 }
                 else
                 {
-                    <a class="link-back" asp-controller="Report" asp-action="List">Back</a>
+                    <a class="link-back" asp-controller="Home" asp-action="Index">Back</a>
                 }
 
+            }
+            else
+            {
+                <a class="link-back" asp-controller="Report" asp-action="List">Back</a>
+            }
+
 
-                @if (Model.Report.Submitted)
+            @if (Model.Report.Submitted)
+            {
+                <legend>
+                    <h1 class="heading-xlarge">Submitted details</h1>
+                </legend>
+            }
+            else
+            {
+                if (Model.IsReadOnly)
                 {
                     <legend>
-                        <h1 class="heading-xlarge">Submitted details</h1>
```

Comment:
what does this change have to do with printing the page?

Reply:
Github weirdness no changes made